FernandoAscencio/theia
# monaco: quick input regression fix

This commit is the first attempt to fix the regression introduced in
 #12095


#### What it does
Closes #12095


#### How to test
1. Run Theia
2. Run the Debugger with `Attach By Process ID`
3. Check if menu options are properly displayed.




#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-Off-By: FernandoAscencio <fernando.ascencio.cama@ericsson.com>
